### DISCUSSION TITLE
Dependabot security scan package registry settings.

### DISCUSSION THREAD

**Discussion by LairdStreak** [community/community#86080] [Code Security] [answered]
### Select Topic Area

Question

### Body

Is there a way to set a private package registries for Dependabot security scans? We have managed to configure this for dependabot package updates but I have not managed to find any documentation on how to use private package registries for the dependabot security scans.

**Comment by carogalvin** [comment]
Are you referring to Dependabot alerts? Dependabot uses your package manifests in repositories to know which packages at which versions you're using, then uses the advisory database to determine if any of them have vulnerabilities. As you've already done, you can then tell Dependabot to use a private package registry to fetch the updates that would patch the vulnerabilities.

Are you looking for a way to scan your private registry to know which of the packages **there** have vulnerabilities and need to be updated?

**Reply by LairdStreak** [reply, marked as answer]
_Are you looking for a way to scan your private registry to know which of the packages there have vulnerabilities and need to be updated_? No :( What I am trying to achieve is to get dependabot security scans to use the package manifest and use the private registries.

**Reply by carogalvin** [reply]
Dependabot already uses the package manifests to detect vulnerabilities and the private registries to detect updates (when configured), so I am confused as to what you are asking for here, could you explain some more?

**Reply by LairdStreak** [reply]
@carogalvin yes that is correct it does use the manifest when normal package update scans run however I have noticed when the security scans run it seems to not take the package manifests into consideration so I am wondering if there is a different setting to use so that it uses the manifest when security scans are run.

**Reply by carogalvin** [reply]
That's odd, what package manager/language is this for?

**Reply by LairdStreak** [reply]
Python. Unfortunately this is a private org repo so I cannot share my file but let me see what happens on the next merge and advise you what I see :) Should be soon.

**Reply by LairdStreak** [reply]
Self Resolved
